code/index.html

```diff
@@ -1,22 +1,163 @@
+  <link rel="stylesheet" href="./style.css"/>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@200;300;400;600;700;900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <!-- meta tag for responsiveness -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+  <header>
+    <!-- Header that changes direction based on screen size -->
+    <div class="nav-container">
+      <div class="logo">
+        <img src="img/Travel..png" width="100px" target="_blank">
+      </div>
+      <div class="navbar">
+        <div class="navbar-item"><a href="#">Home</a></div>
+        <div class="navbar-item"><a href="#">Guides</a></div>
+        <div class="navbar-item"><a href="#">Top Picks</a></div>
+        <div class="navbar-item"><a href="#">Destinations</a></div>
+      </div>
+    </div>
+  </header>
+
+  <div class="main-container">
+    <!-- Hero section with header and 2 column flexbox with gap -->
+    <div class="hero">
+      <h1>Find your inspiration for your next holiday.</h1>
+      <div class="hero-children">
+        <div class="hero-p">
+          <p>
+            If you’re anything like me you’ll have a never ending wish list of destinations and experiences to tap into
+            for your holiday inspiration. If not, you’re in luck because we've got you covered with a diverse selection
+            of
+            short breaks, experience travel and holidays to whet your wanderlust.
+          </p>
+        </div>
+        <div class="hero-p">
+          <p>
+            If you’re anything like me you’ll have a never ending wish list of destinations and experiences to tap into
+            for your holiday inspiration. If not, you’re in luck because we've got you covered with a diverse selection
+            of
+            short breaks, experience travel and holidays to whet your wanderlust.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <!-- Flexbox with 2 items that change order when resized -->
+    <section class="featured-article">
+      <div class="flex-container-2">
+        <article>
+          <h2>
+            Ideas to plan for a great trip
+          </h2>
+          <p>
+            Planning a trip overseas, whether long or short, can seem overwhelming. But it doesn’t have to be! In this
+            article, we’ll show you how to take the stress out of planning a trip, by laying out a simple step-by-step
+            guide that’ll ensure you don’t forget anything. </p>
+          <button class="featured-article-button">Read More</button>
+        </article>
+      </div>
+      <div class="flex-container-2">
+        <img src="img/rolands-varsbergs-miKmVyq3qhE-unsplash.jpg" alt="article-1">
```

Comment:
Maybe name of `alt` should explain what is in the picture for someone who cannot see it?

Reply:
Very good comment. I guess I was a bit lazy with accessibility 🙄